includes/models/model.llms.course.php

```diff
@@ -663,6 +697,42 @@ public function is_enrollment_open() {
 
 	}
 
+	/**
+	 * Returns a message if the enrollment period has not started, the enrollment period has ended,
+	 * or there is no more enrollment capacity, else returns false.
+	 *
+	 * @since [version]
+	 *
+	 * @return false|string
+	 */
+	public function is_enrollment_restricted() {
+
+		$restricted_message = false;
+		if ( ! $this->has_enrollment_period_started() ) {
+			$restricted_message = $this->get( 'enrollment_opens_message' );
+		}
+		if ( $this->has_enrollment_period_ended() ) {
+			$restricted_message = $this->get( 'enrollment_closed_message' );
+		}
+		if ( ! $this->has_capacity() ) {
+			$restricted_message = $this->get( 'capacity_message' );
+		}
```

Comment:
I think the behavior here is different than previously found in the `not-purchaseable` template.

The first two conditions change from an if / elseif to an if / if, which doesn't really matter but the last condition could be true in addition to one of the two enrollment period conditions.

The template *could* have two different messages shown -- one for one of the two enrollment period conditions and a second message if the course capacity is also exhausted.

After this change though the first message would get lost and only the latter (capacity) message would be displayed.